cmd/yurt-controller-manager/app/controllermanager.go

```diff
@@ -305,7 +305,7 @@ func NewControllerInitializers() map[string]InitFunc {
 	controllers := map[string]InitFunc{}
 	controllers["nodelifecycle"] = startNodeLifecycleController
 	controllers["yurtcsrapprover"] = startYurtCSRApproverController
-
+	controllers["podupgrade"] = startPodUpgradeController
```

Comment:
how about rename `podupgrade` to `daemonpodupdater`?

Reply:
OK. I'm trying separate this controller into a base controller and a daemonset controller as your advice. I'll modify the name then.